src/classes/BDI_DataImportService.cls

```diff
@@ -691,9 +691,10 @@ global with sharing class BDI_DataImportService {
         String paymentStatus = dataImport.Payment_Status__c;
 
         return String.isBlank(paymentStatus) ||
-            dataImport.Payment_Status__c.equalsIgnoreCase(
+            (dataImport.Payment_Status__c.equalsIgnoreCase(
                 GE_PaymentServices.ElevateTransactionStatus.CAPTURED.name()
-            );
+            ) || (dataImport.Payment_Method__c == ACH_PAYMENT_METHOD
+                && dataImport.Payment_Elevate_ID__c != null));
```

Comment:
It looks like the comment above this method needs to be updated.  It currently begins with "Checks a collection of data imports..." but the function only takes a single DI as a param.

Also, does the logic mean that "isSuccessfulPayment" returns true when the Payment_Status field is blank?  Is that accurate?  

Can any of these predicates be extracted into their own function/expression to make this a bit more readable?

Reply:
This change was lifted from the #6108 since it is supposed to be a change from the CCS team. I added this so DIs with ACH transactions get processed. I suspect that this line will change once that PR merges in.